cmd/admin/cert.go

```diff
@@ -221,6 +224,23 @@ func (a *admin) serialsFromRegID(ctx context.Context, regID int64) ([]string, er
 	return serials, nil
 }
 
+func (a *admin) checkValidSerial(serial string) (string, error) {
```

Comment:
Real comment: This function doesn't need to be a method; it doesn't rely on access to any of the `admin` struct's fields.

Tiny nitpick: A function whose name starts with "check" suggests to me that it will return _only_ an error. This function also returns the cleaned-up version of the serial. I'd suggest a name like "cleanSerial" or "canonicalSerial".
```suggestion
func cleanSerial(serial string) (string, error) {
```